openapi/openapi.yaml

```diff
@@ -244,6 +342,31 @@ components:
       required:
         - id
         - label
+    ZoningDistrictClass:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The code associated with the Zoning class.
+          # format: 
+          example: M1
+        category:
+          type: string
+          enum:
+            - "Residential"
+            - "Commercial"
+            - "Manufacturing"
+          description: The type of zoning district.
+          # format:
+          # example:
+        description:
+          type: string
+          description: Zoning class descriptions from Zola.
+        color:
+          type: string
+          description: Zoning classes from the existing Zola layer groups.
```

Comment:
Let's keep references to Zola out of these descriptions. They should stand on their own. Can be as simple as "the color for zoning district class"